kyuubi-server/src/main/scala/org/apache/kyuubi/sql/plan/command/RunnableCommand.scala

```diff
@@ -30,6 +30,10 @@ trait RunnableCommand extends KyuubiTreeNode {
 
   protected var isAdmin: Boolean = false
 
+  val statement: String
```

Comment:
Why is the new `statement` attribute needed? It seems that the `name() `method is enough.

Reply:
`Execute xxx Command` sounds good, I will fix it in another issue.